src/Modules/OrchardCore.Commerce.Payment/Controllers/PaymentBaseController.cs

```diff
@@ -19,6 +20,10 @@ protected PaymentBaseController(INotifier notifier, ILogger logger)
         _logger = logger;
     }
 
+    [SuppressMessage(
+        "Critical Code Smell",
+        "S6967:ModelState.IsValid should be called in controller actions",
+        Justification = "This is not a controller action.")]
```

Comment:
It should rather be `protected`, then.

Reply:
I thought it was used from another class in testing, but I was mistaken.